client/src/utils/bondingcurveCalculator.js

```diff
@@ -0,0 +1,98 @@
+import Web3 from 'web3';
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['Web3' is defined but never used.](https://app.codacy.com/manual/oceanprotocol/commons/pullRequest?prid=5420705)